__tests__/workers/userCreatedCio.ts

```diff
@@ -0,0 +1,60 @@
+import nock from 'nock';
+import worker from '../../src/workers/userCreatedCio';
+import { ChangeObject } from '../../src/types';
+import { expectSuccessfulTypedBackground } from '../helpers';
+import { User } from '../../src/entity';
+import { PubSubSchema } from '../../src/common';
+import { cio } from '../../src/cio';
+
+jest.mock('../../src/cio', () => ({
+  ...(jest.requireActual('../../src/cio') as Record<string, unknown>),
+  cio: { identify: jest.fn() },
+}));
+
+beforeEach(async () => {
+  jest.clearAllMocks();
+  nock.cleanAll();
+});
+
+describe('userCreatedCio', () => {
```

Comment:
If you can add this kind of test https://github.com/dailydotdev/daily-api/blob/45364867794974b43f4713e3fcf362da2eb27f65/__tests__/workers/commentDownvotedRep.ts#L53-L59 for new CIO workers, just to make sure they are actually registered in `workers/index`.

Reply:
i missed both of your comments. i'll fix that in a follow up PR